improver/nowcasting/accumulation.py

```diff
@@ -232,11 +245,12 @@ def _get_cube_subsets(self, cubes, forecast_period):
         # If the input is a numpy array, get the integer value from the array
         # for use in the constraint.
         if isinstance(forecast_period, np.ndarray):
-            forecast_period, = forecast_period
+            (forecast_period,) = forecast_period
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assigning the same variable 'forecast_period' to itself](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5385606)